tests/integration/replication.tcl

```diff
@@ -575,7 +575,7 @@ start_server {tags {"repl"}} {
     $master config set rdbcompression no
     # If running on Linux, we also measure utime/stime to detect possible I/O handling issues
     set os [catch {exec unamee}]
-    set measure_time [expr {$os == "Linux"} ? 1 : 0]
+    set measure_time [expr {$os == "Linux" && !$::no_latency} ? 1 : 0]
```

Comment:
I would rather just make the assertions conditional, and keep the verbose logging